openapi_python_client/templates/client.py.jinja

```diff
@@ -92,6 +105,7 @@ class Client:
                 cookies=self._cookies,
                 headers=self._headers,
                 timeout=self._timeout,
+                transport=self._transport,
```

Comment:
If `retries` were passed to `__init__`, `self._transport` would still be `None` here and have no effect. If we keep `self._retries` (and don't remove it, as an earlier comment suggests), we'd need to solve this somehow, maybe with a `self._get_transport()` method.